.github/workflows/test-inductor.yml

```diff
@@ -18,6 +18,7 @@ jobs:
     - run: make setup_nightly
     - run: make develop
     - run: pytest ./test/test_torchinductor.py
+    - run: pytest ./test/test_torchinductor_opinfo.py
```

Comment:
You need to add a CircleCI task as well, https://github.com/pytorch/torchdynamo/blob/main/.circleci/config.yml . We are not running GPU instance for GHA yet.

Reply:
This is already ran in CircleCI, see https://app.circleci.com/pipelines/github/pytorch/torchdynamo/2764/workflows/67bcf48e-3722-4216-ab6d-fb487b7e97b3/jobs/19887

In CircleCI, it runs pytest on root . All tests will be automatically included.

Reply:
NVM, you are right.